lambdas/meetupcom/serverless.yml

```diff
@@ -9,12 +9,17 @@ provider:
     - Effect: Allow
       Action:
         - s3:*
+      Resource: "*"
```

Comment:
We can probably refine this more in the future, but just providing `/*` on the bucket didn't provide listing permissions for sone reason, which is required to pick out the group file.

Reply:
Once everything is deployed we can revisit the permissions to lock them down. Added #491 as a placeholder to remember to revisit this later